src/api/statements/LodexRunQuery.js

```diff
@@ -13,7 +13,7 @@ export const createFunction = mongoClientImpl =>
         }
         const from = this.getParam('from', data.$from || 'dataset');
         const query = this.getParam('query', data.query || {});
-        const limit = this.getParam('limit', data.limit || 10);
+        const limit = this.getParam('limit', data.limit || 1000000);
```

Comment:
Is this the real default value or just to debug?
If you confirm it's the correct value, I'll merge! :)

Reply:
yes, it's the correct value